Patcher/Program.cs

```diff
@@ -204,28 +210,37 @@ internal static void Main(string[] args)
                     if (backupFileInfo.Exists)
                         Console.WriteLine($"Backup '{backupFileInfo.Name}' created.");
 
-                    Console.WriteLine("Searching for theme offset..");
+                    Console.WriteLine("Searching for theme offset...");
 
                     var buffer = ms.ToArray();
+
                     var lightOffsets = FindPattern(_lightPattern, buffer).ToList();
                     var darkOffsets = FindPattern(_darkPattern, buffer).ToList();
                     lightOffsets.AddRange(darkOffsets);
 
                     var found = lightOffsets.Any();
-
                     if (!found)
                     {
-                        Console.WriteLine("Could not find the theme offset in the specified executable.");
+                        Console.WriteLine("Error: Could not find the theme offset in the specified executable!");
                         return;
                     }
-
-                    var themeByte = themeName == "dark" ? (byte)0x75 : (byte)0x74;
+                    var foundMoreThanExpected = lightOffsets.Count > 1;
+                    if (foundMoreThanExpected)
+                    {
+                        Console.WriteLine("Warning: Found more than expected theme offsets. Outcome might be unexpected!");
```

Comment:
This would break the previous version for MacOS, which (even though it finds multiple instances of the pattern) still works as expected.

As far as I know, this currently only occurs on the MacOS binary.